pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -118,10 +118,31 @@ type BundleTarget struct {
 type AdditionalFormats struct {
 	// JKS requests a JKS-formatted binary trust bundle to be written to the target.
 	// The bundle is created with the hardcoded password "changeit".
-	JKS *KeySelector `json:"jks,omitempty"`
+	JKS *JKS `json:"jks,omitempty"`
 	// PKCS12 requests a PKCS12-formatted binary trust bundle to be written to the target.
 	// The bundle is created without a password.
-	PKCS12 *KeySelector `json:"pkcs12,omitempty"`
+	PKCS12 *PKCS12 `json:"pkcs12,omitempty"`
+}
+
+type JKS struct {
+	KeySelector `json:",inline"`
+
+	// Password to JKS trust store
+	//+optional
+	//+kubebuilder:validation:MinLength=1
+	//+kubebuilder:validation:MaxLength=30
```

Comment:
suggestion: It's good to have _some_ max length, but I think 30 might be a bit small. A reasonable password to use here would be 32 hex encoded bytes (`openssl rand -hex 32`), which would have a length of 64 bytes. I'd suggest setting the max length to 128 so that there is a max but if someone wants to use a really secure password they can.

Reply:
Fixed.